Cargo.lock

```diff
@@ -2,13 +2,23 @@
 # It is not intended for manual editing.
 version = 3
 
+[[package]]
+name = "Inflector"
+version = "0.11.4"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "fe438c63458706e03479442743baae6c88256498e6431708f6dfc520a26515d3"
+dependencies = [
+ "lazy_static",
+ "regex",
+]
+
 [[package]]
 name = "actix-codec"
-version = "0.5.1"
+version = "0.5.2"
```

Comment:
Looks like you've run `cargo update`.
Unfortunately, I have to ask you to roll this change back, as we have several crypto dependencies specified via git, and `cargo update` may pin them to newer commits, which may be breaking (e.g. the node can start producing different state diffs or have other unexpected consequences).